packages/webapp/public/locales/en/translation.json

```diff
@@ -620,26 +577,18 @@
       "BODY": "We’ll notify you once your upload is complete and your sensors have been created. Feel free to navigate away from this window.",
       "TITLE": "This is taking longer than expected..."
     },
-    "CEREMONIAL_AREA": {
-      "EDIT_TITLE": "Edit ceremonial area",
-      "NAME": "Ceremonial area name",
-      "TITLE": "Add ceremonial area"
-    },
     "CONFIRM_RETIRE": {
       "BODY": "Retiring this location will remove it from the farm map.",
       "TITLE": "Retire location?"
     },
     "DRAWING_MANAGER": {
-      "REDRAW": "Redraw",
-      "ZERO_AREA_DETECTED": "Field with no area detected. Please add more points to current drawing or draw again.",
```

Comment:
Ternary in the call to `t()` breaks script too apparently:

```
{t(
  showZeroAreaWarning
    ? 'FARM_MAP.DRAWING_MANAGER.ZERO_AREA_DETECTED'
    : 'FARM_MAP.DRAWING_MANAGER.ZERO_LENGTH_DETECTED',
)}
```

<img width="445" alt="Screenshot 2023-11-10 at 6 17 33 PM" src="https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/f4e58b87-a54e-4d04-bf5e-47b8a174cb01">